src/librustdoc/html/render/write_shared.rs

```diff
@@ -304,7 +304,12 @@ pub(super) fn write_shared(
     }
 
     if cx.shared.layout.scrape_examples_extension {
-        write_minify("scrape-examples.js", static_files::SCRAPE_EXAMPLES_JS, cx, options)?;
+        cx.write_minify(
+            SharedResource::Unversioned { name: "scrape-examples.js" },
```

Comment:
Sorry, I should have been more clear - this changes based on the crate currently being compiled, so it needs to use `CrateSpecific`.

Reply:
Oh good point, I misunderstood the options.